### ISSUE TITLE
[Bug]: waiting for initialization of module: Descriptives

### ISSUE THREAD

**Issue by purushottamb** [jasp-stats/jasp-issues#1862]
### JASP Version

0.16.3

### JASP Module

jaspDescriptives

### What OS are you seeing the problem on?

Windows 10

### Bug Description

Descriptive Statistics module showing a message
"Waiting for Initialisation of Module: Descriptives"

### Expected Behaviour

should have reported descriptive statistics

### Steps to Reproduce

1. Load data
2. Check Descriptives
3. 
4. 
...

### Final Checklist

- [X] I have included a screenshot showcasing the issue, if any.
- [X] I have included a JASP file that causes the crash/bug, if any.
- [X] I have accurately described the bug, and steps to reproduce it.

**Comment by tomtomme** [comment]
On a first start this may need a minute. How long did you wait? How big was the dataset?

**Comment by purushottamb** [comment]
I thought similar and waited for long and this was not first start.
I copied descriptive with ‘+’ button to generate tables for different variables.
Then it took forever to load with “waiting for initialization of module: Descriptives” message.
The data wasn’t that big. About 700 observations.

I relaunched JASP and it loaded.

PS:  “+” button regenerated the issue once.

Working fine now.

Thank you

-
Puru

Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows
